cli/command/image/tree.go

```diff
@@ -281,11 +281,23 @@ func printNames(out *streams.Out, headers []imgColumn, img topImage, color, unta
 		_, _ = fmt.Fprint(out, headers[0].Print(untaggedColor, "<untagged>"))
 	}
 
-	for nameIdx, name := range img.Names {
-		if nameIdx != 0 {
-			_, _ = fmt.Fprintln(out, "")
+	// TODO: Replace with namesLongestToShortest := slices.SortedFunc(slices.Values(img.Names))
+	// once we move to Go 1.23.
+	namesLongestToShortest := make([]string, len(img.Names))
+	copy(namesLongestToShortest, img.Names)
```

Comment:
just out of interest, why do we need to copy the `img.Names` slice? updating the slice in-place by `sort.Slice` shouldn't have a negative effect here? e.g. another place accessing `img.Names` shouldn't care if `img.Names` was sorted or not?

Reply:
Yeah we don't really need it in this context, but I think it's also really not a nice thing for the `printNames` function to mutate the input it got 😄 

It will also avoid any future errors if this got refactored and somehow ended up being executed in different context.